system-int-test/test/cmr/system_int_test/ingest/granule_validation_test.clj

```diff
@@ -431,3 +428,72 @@
   ;; Turn it back on after the test
   (dev-sys-util/eval-in-dev-sys
    `(cmr.metadata-db.services.concept-constraints/set-enforce-granule-ur-constraint! true)))
+
+(def ^:private coll-file
+  "temp file for collection metadata in multipart request"
+  "cmr_5260_collection.xml")
+
+(def ^:private granule-file
+  "temp file for granule metadata in multipart request"
+  "cmr_5260_granule.json")
+
+(defn- validate-granule-multipart
+  "Returns the granule validataion result for the given collection and granule
+  via multipart granule validation request."
+  [granule-concept coll-concept]
+  ;; clj-http does not work with content-type that contains semicolons
+  ;; (e.g. application/vnd.nasa.cmr.umm+json; version=1.4)
+  ;; So we execute curl command directly from shell for UMM-G granule validation requests
+  (try
+    (spit coll-file (:metadata coll-concept))
+    (spit granule-file (:metadata granule-concept))
+    (let [{:keys [provider-id native-id]} granule-concept
+          validate-url (url/validate-url provider-id :granule native-id)
+          coll-form (format "collection=<%s;type=%s" coll-file (:format coll-concept))
+          granule-form (format "granule=<%s;type=%s" granule-file (:format granule-concept))
+          {:keys [out]} (shell/sh "curl" "-i" "-F" granule-form "-F" coll-form validate-url)]
```

Comment:
This adds a dependency on `curl` for our tests, but I guess that's ok. We know that our CI environment has curl installed.

Reply:
Yep, we are already using curl is some other tests.